app/page.tsx

```diff
@@ -199,6 +186,19 @@ function AreaPanel({
   );
 }
 
+// Helper function to get consistent color mapping
+function getColorKeyForModel(modelName: string, fallbackIndex: number): keyof typeof colors.lightText {
+  if (modelName.includes("Qwen2")) return "blue";
+  if (modelName.includes("DeepSeek")) return "green";
+  if (modelName.includes("QWQ")) return "yellow";
```

Comment:
this will never match, `includes` is case sensitive
why do we even have this function? now we will have to manage the colors manually, why don't just use index?